public/javascripts/continuousPlay.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-unused-vars */
+
+//page numeration
+let pageNum = parseInt(localStorage.getItem("pageNum"));
+if (pageNum) {
+  console.log("there is a page num");
+  document.getElementById("page-counter").innerText = `${pageNum} of 20`;
+} else {
+  console.log("there is NO page num");
+  document.getElementById("page-counter").innerText = "1 of 20";
+  localStorage.setItem("pageNum", "1");
+  pageNum = 1;
+}
+if (pageNum === 5) {
+  document.getElementsById("button-next").innerText = "Finish";
+}
+
+//on clicking next
+function nextGuess() {
+  const guesses = document.querySelectorAll('input[name="guess"]');
+  const gameName = document.getElementById("game-name").innerText;
+
+  //saving currect guess data
+  localStorage.setItem(`${pageNum}.gameName`, gameName);
```

Comment:
Ok, so as talked let's try to save guessing data for now as a JSON object instead of separate entries.